core/i18n.py

```diff
@@ -0,0 +1,27 @@
+import os
+import json
+
+
+class Response:
+    def __init__(self, directory, language, prefix):
+        self.directory = directory
+        self.language = language
+        self.prefix = prefix
+        self.responses = self.load()
+
+    def load(self):
+        data = {}
+        for file in os.listdir(self.directory):
+            if file.endswith(".json"):
+                with open(f"{self.directory}/{file}", encoding="utf-8") as f:
+                    data[file.rstrip(".json")] = json.load(f)
+        return data
+
+    def get(self, item):
+        try:
+            response = self.responses[self.language][item]
+        except KeyError:
+            response = self.responses["en"][item]
+            print(f"Could not find a translation ({self.language}) for the requested i18n item: {item}. Please file an issue on GitHub.")
+        response = response.replace("{prefix}", self.prefix)
```

Comment:
Feels like this should be handled in the `.format` elsewhere, seems odd to only handle this replacement here.

Reply:
Wouldn't it feel redundant to .format(prefix) everywhere?

Reply:
True, although I don't like the concept of one variable being handled differently from the others, might get confusing down the line

Reply:
Yeah, I can see your point too.
I will leave this unresolved for now while I think about it.